_pages/dev/components/shared-components/media-component.md

```diff
@@ -0,0 +1,99 @@
+
+Different product images for different screen sizes are typically generated based on the same master image. This results in a media container that holds a number of media items that only differ in pixel size, but are equal in terms of content and proportions.
+
+The media formats for product images can be configured in both the back end and the front end. The following list shows the (default) formats that are used in the sample data and in the Spartacus configuration:
+
+- cartIcon
+- thumbnail
+- product
+- zoom
+
+### Content Images
+
+Content images are driven by CMS component data. Media items are used in different banner components. Unlike product images, banner images are often manually optimized for various screen sizes. This means that a content manager uploads alternative images to the banner for each media format. The technical structure is the same for product and banner media, but the formats are different. The following list shows the (default) formats that are used in the sample data and in the Spartacus configuration:
+
+- mobile
+- tablet
+- desktop
+- widescreen
+
+## Localized Media
+
+The back end supports localized media, which means that different media items can be used for different languages. This is sometimes used for content images that contain localized text.
+
+Localized media works transparently for the media component. Whenever the size context changes in Spartacus (including for languages), the CMS and product data are cleared from the state, which results in a reload of the data for the given context.
+
+## Implementation Details
+
+The `cx-media` media component renders images with the native `img` HTML element. To support an optimized image for the given element, a container with multiple images is expected. The various images in the container are evaluated by their media format and compared to a media format configuration in Spartacus.
+
+The native `img` element supports multiple images with the `srcset` attribute. This attribute takes various image sources and combines them with an associated width descriptor. The browser selects and downloads the correct image based on which dimensions of the img element are used. With this approach, you do not need to provide a specific format for the media component, although you can do this with the `format` input.
+
+The `srcset` attribute also supports the pixel density descriptor, but it is (currently) not supported in Spartacus. This pixel density descriptor can be used to select different images for different devices. For example, an image width descriptor of 400 px might be rendered on retina devices at a maximum of 200 px because these devices double the pixels to provide an optimized image resolution for their device screens.
+
+The mapping from an image format to the `srcset` width descriptor is driven by the media configuration in Spartacus. The given media container
+
```

Comment:
The mapping from an image format to the `srcset` width descriptor is driven by the media configuration in Spartacus. The given media container...

It looks like the second sentence here is incomplete.

Reply:
👍